configure.ac

```diff
@@ -532,6 +533,45 @@ AS_IF([test "x${enable_opus_cdda}" != "xno"],
 
 AM_CONDITIONAL(USE_OPUS, test "${HAVE_OPUS}" = "yes")
 
+
+dnl FluidSynth software MIDI synthesiser
+dnl ------------------------------------
+dnl
+AH_TEMPLATE(C_FLUIDSYNTH, [Define to 1 to enable FluidSynth MIDI synthesiser])
+AC_ARG_ENABLE(fluidsynth,
+              AS_HELP_STRING([--disable-fluidsynth],
+                             [Disable FluidSynth integration]))
+
+if test "${enable_fluidsynth}" != "no" ; then
+    AC_MSG_CHECKING(for FluidSynth >= 2.0.0)
+    AC_COMPILE_IFELSE(
+        [AC_LANG_SOURCE([
+            #include <fluidsynth.h>
+            constexpr int version_num(int x, int y, int z) {
+                return x * 10000 + y * 100 + z;
+            }
+            constexpr int lib_version = version_num(FLUIDSYNTH_VERSION_MAJOR,
+                                                    FLUIDSYNTH_VERSION_MINOR,
+                                                    FLUIDSYNTH_VERSION_MICRO);
+            static_assert(lib_version >= version_num(2, 0, 0));
```

Comment:
If we need 2.*.* would FLUIDSYNTH_VERSION_MAJOR >= 2 cover all the cases?
(not that I have any issues with the current code, which is handy if we need to slice and dice versions more carefully in the future :-)

Reply:
Ah; the code looks like this, because I started with minimum FluidSynth version 2.1.0, and then for some reason, I decided 2.0.0 is good enough.